Sources/Sentry/Public/SentryHub.h

```diff
@@ -41,6 +41,8 @@ SENTRY_NO_INIT
 @property (nonatomic, strong)
     NSMutableArray<NSObject<SentryIntegrationProtocol> *> *installedIntegrations;
 
+@property (nonatomic, strong) NSMutableArray<NSString *> *installedIntegrationNames;
```

Comment:
Instead of always recreating the names from the `installedIntegrations` array (which contains actual instances), we keep this new array around. We could get rid of it but I don't think it's worth it. 

This is the direct replacement of `SentryOptions.enabledIntegrations`.

Reply:
m: This should at least be read only. But I think it would be better for this cache to be private. Why do you think we need this property for SentryHub?

Reply:
You're right, this should indeed be private.
I added it to SentryHub because `installedIntegrations` is also there, and it makes sense here.

Reply:
It's actually very strange that `installedIntegrations` is public and writable as well.

Reply:
Both are now private